src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
```

Comment:
Are the tests passing? How is the test suite started? (thinking about setting up some CI)

Reply:
As far as I can tell, all the tests are failing.

I haven't looked at the test cases much, but I'd like to get them running again.
I'll take a look when I start moving the logout button.

You should be able to run `yarn test`; you'll get an interactive command-line test runner.

Any preferences for a CI platform? I can start setting that up.

Reply:
Thanks for the explanation, glad to hear you have a rough plan to get tests working again.

>Any preferences for a CI platform? I can start setting that up.

I have the most experience with TravisCI, and found it pretty flexible (root access and docker support and great free plan for FOSS), but I'm not attached if there's something you're more comfortable with.